client/src/pages/PostPage.js

```diff
@@ -1,17 +1,20 @@
 import React, { useEffect, useReducer } from "react";
+import { useHistory, useParams } from "react-router-dom";
 import axios from "axios";
 import { Modal } from "antd";
-import { useHistory, useParams } from "react-router-dom";
 
+// Local
+import Post, { CONTENT_LENGTH } from "components/Feed/Post";
 import EditPost from "components/CreatePost/EditPost";
+import { typeToTag } from "assets/data/formToPostMappings";
 import Loader from "components/Feed/StyledLoader";
 import { FEED } from "templates/RouteWithSubRoutes";
```

Comment:
```suggestion
import { FEED, LOGIN } from "templates/RouteWithSubRoutes";
```

Something must have happened with merge conflict resolution? I get this

```
client             | [start]   Line 17:10:  Parsing error: Identifier 'FEED' has already been declared
client             | [start] 
client             | [start]   15 | 
client             | [start]   16 | // Constants
client             | [start] > 17 | import { FEED, LOGIN } from "templates/RouteWithSubRoutes";
```

Reply:
Done.